examples/inference_restoring_model_dependencies/README.md

```diff
@@ -0,0 +1,20 @@
+### MLflow evaluation Examples
```

Comment:
Can we make this examples directory / README more generic?

I think we should call it `examples/spark_udf` and tell users in the README that the purpose is to show examples of using Spark UDF for batch inference, including environment reproducibility capabilities with `env_manager="conda"`.